python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/ethernet-interfaces.j2

```diff
@@ -334,6 +334,11 @@
 | {{ link_tracking_interface.name }} | {{ link_tracking_group.name }} | {{ link_tracking_group.direction }} |
 {%                 endif %}
 {%             endfor %}
+{%             if link_tracking_interface.link_tracking.direction is arista.avd.defined %}
+{%                 for group_name in link_tracking_interface.link_tracking.groups | arista.avd.natural_sort %}
+| {{ link_tracking_interface.name }} | {{ group_name }} | {{ link_tracking_interface.link_tracking.direction }} |
```

Comment:
i think in new model we should join all group by comma and keep in one row only.
cc: @ClausHolbechArista

Reply:
sure

Reply:
Changed!